cleaning/pdf-mining/pdftotxt.py

```diff
@@ -29,11 +29,42 @@ def parse_statement_date(line, iterator):
                 groups = re.split(r'\s{2,}', line)
                 if not groups:
                     return
-                statement_date = dateparser.parse(groups[0], languages=LANGUAGES)
+                statement_date = parse_date(groups[0])
                 if statement_date:
                     return statement_date
 
 
+# "(1,380.77)" and "1,380.77 CR" will translate to -1380.77
+def parse_amount(amount_str):
+    try:
+        amount_str = amount_str.lower().strip().replace(',', '')
+        cleaned = re.sub(r'\(|\)|cr', '', amount_str).strip()
```

Comment:
I was just checking if the value contains `(` and turns it into negative 😆

Reply:
(123.12) is what OCBC uses. DBS and UOB use a different format: 123.12 CR